tests/codeception/acceptance/StanfordPersonImporterCest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Test the importer works.
+ */
+class StanfordPersonImporterCest {
+
+  /**
+   * Add credentials from environment variables and verify content is imported.
+   */
+  public function testImporter(AcceptanceTester $I) {
+    $I->runDrush('pm:enable stanford_person_importer');
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/taxonomy/manage/cap_org_codes/add');
+    $I->fillField('Name', 'Web Services');
+    $I->fillField('su_cap_org_code[0][value]', 'BSWS');
+    $I->click('Save');
+
+    $I->amOnPage('/admin/config/importers/person-importer');
+    $I->fillField('CAP Username', getenv('CAP_USERNAME'));
+    $I->fillField('CAP Password', getenv('CAP_PASSWORD'));
```

Comment:
I used to put valid credentials in the default `.env` file for our Lando devs so I suspect I should do this again as this will fail for most of our devs on local unexpectedly. Should we start packaging a `.env` file for those with a custom/native local build as well?

Reply:
yeah this allows for credentials to be saved on circle ci